.eleventy.js

```diff
@@ -42,17 +49,17 @@ module.exports = function (config) {
 
     // Custom collections
     const livePosts = post => post.date <= now && !post.data.draft;
-    config.addCollection("posts", collection => {
-        return [
-            ...collection.getFilteredByGlob("./src/posts/*.md").filter(livePosts)
-        ];
-    });
-
-    // The following collection is used to create a collection of posts for the RSS feed.
-    config.addCollection("postFeed", collection => {
-        return [...collection.getFilteredByGlob("./src/posts/*.md").filter(livePosts)]
-            .reverse()
-            .slice(0, site.maxPostsInFeed);
+    Object.keys(siteConfig.languages).forEach(lang => {
```

Comment:
Probably somewhere before this there should be some validation that the `defaultLanguage` exists in the set of configured languages, and throw an appropriate error if not.

Reply:
I've added an error to the output to explain other build problems that might arise if this is misconfigured.